<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoControlBar</class>
 <widget class="QWidget" name="VideoControlBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>585</width>
    <height>221</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(160, 160, 160,0.1);</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>8</number>
   </property>
   <property name="leftMargin">
    <number>10</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>10</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QPushButton" name="play_pause_btn">
     <property name="text">
      <string>播放</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="mute_btn">
     <property name="text">
      <string>静音</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSlider" name="volume_slider">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::NoTicks</enum>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="fresh_btn">
     <property name="text">
      <string>刷新</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="max_screen_btn">
     <property name="text">
      <string>最大化</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="full_screen_btn">
     <property name="text">
      <string>全屏</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
